iothub/service/tests/ClientWebSocketTransportTests.cs

```diff
@@ -29,17 +29,17 @@ public class ClientWebSocketTransportTests
         private static HttpListener s_listener;
         private static byte[] s_byteArray = new byte[10] { 0x5, 0x6, 0x7, 0x8, 0xA, 0xB, 0xC, 0xD, 0xE, 0xF };
 
-        [AssemblyInitialize()]
```

Comment:
Why would this be required for all unit tests in the assembly?

Also, use the async version.